website/pages/docs/runtime/interpolation.mdx

```diff
@@ -58,7 +58,7 @@ task "docs" {
 }
 ```
 
-## Node Variables [](#interpreted_node_vars)
+## Node Variables ((#interpreted_node_vars, #node-variables-))
```

Comment:
I love the **anchor link aliases**. Thank you for documenting this in the PR.

**NIT**: I wasn’t clear whether one of these was the **primary** link. I now see that the primary link is generated from the heading.